tools/setupResourceScripts/README.md

```diff
@@ -31,3 +31,42 @@ in the "integration 4" environment using a new billing profile.
+   * If you do not already have a managed application created, follow instructions in jade-getting-started.md
+   * Make note of the email account that set during managed app setup, you'll need to authenticate with this user
+   * Make note of the "Application name" set during managed app setup. You'll pass this in with the --azure_application_name flag
+   * Set the --billing_profile_file_name flag to the dev azure file: `--billing_profile_file_name='./files/azure_billing_profile_dev.json'`
+2. Determine TDR instance you want to populate (i.e.local, BEE, etc.)
+   * If running locally, set the --host flag to `http://localhost:8080`. Get TDR instance running locally by following instructions in jade-getting-started.md.
+   * Otherwise, set the --host flag to the appropriate TDR instance.
+3. If needed, set up your python instance:
+   * `python3 -m venv c:/path/to/myenv`
+   * `source c:/path/to/myenv/bin/activate`
+4. Authenticate with the email account you set up during managed app setup:
+   * `gcloud auth login <email>`
+5. Run the python script
+   * `cd jade-data-repo/tools/setupResourceScripts`
+   * `pip3 install -r requirements.txt`
+   * Run the set up script including all of the arguments, plus pointing to the azure OMOP dataset file:
+   * `python3 setup_tdr_resources.py --host <datarepo_url> --datasets='./files/azure_omop_dataset.json' --azure_application_name <azure_application_name> --billing_profile_file_name './files/azure_billing_profile_dev.json'`
+   * If you have a billing account already created, you can use the `--azure_profile_id` flag to pass in the billing profile id, then you no longer need to pass in some of the other flags:
+   * `python3 setup_tdr_resources.py --host <datarepo_url> --datasets='./files/azure_omop_dataset.json' --azure_profile_id <profile_id>`
+
+### GCP OMOP Dataset and Snapshot Builder Settings Setup
+1. Determine TDR instance you want to populate (i.e.local, BEE, etc.)
+  * If running locally, set the --host flag to `http://localhost:8080`. Get TDR instance running locally by following instructions in jade-getting-started.md.
+  * Otherwise, set the --host flag to the appropriate TDR instance.
+2. If needed, set up your python instance:
+  * `python3 -m venv c:/path/to/myenv`
+  * `source c:/path/to/myenv/bin/activate`
+3. Authenticate with the email account that has access to the GCP billing account `00708C-45D19D-27AAFA`:
+  * `gcloud auth login <email>`
+4. Run the python script
+  * `cd jade-data-repo/tools/setupResourceScripts`
+  * `pip3 install -r requirements.txt`
+  * Run the setup script including all the arguments, plus pointing to the GCP OMOP dataset file:
+  * `python3 setup_tdr_resources.py --host <datarepo_url> --datasets='./files/gcp_omop_dataset.json'`
+  * If you have a billing account already created, you can use the `--gcp_profile_id` flag to pass in the billing profile id.
```

Comment:
Note to reviewers: These instructions cover [DC-930 - Write instructions for getting on Azure on TDR - setting up local OMOP DB](https://broadworkbench.atlassian.net/browse/DC-930)